cmd/iam.go

```diff
@@ -1691,10 +1700,16 @@ func (sys *IAMSys) NormalizeLDAPMappingImport(ctx context.Context, isGroup bool,
 		mappingValue := policyMap[origKeys[0]]
 		delete(policyMap, origKeys[0])
 		policyMap[normKey] = mappingValue
+		iamLogIf(ctx, fmt.Errorf("(import-info) normalized LDAP DN mapping from `%s` to `%s`", origKeys[0], normKey))
```

Comment:
```suggestion
		iamLogIf(ctx, fmt.Errorf("(import-info) normalized LDAP DN mapping from `%s` to `%s`", origKeys[0], normKey))
```

this is just informational right? for this iamLogEvent just like rebalanceLogEvent()

Reply:
yes, this is logger.InfoKind, you can add iamInfo() function @donatello